skyportal/handlers/api/gcn.py

```diff
@@ -450,12 +450,19 @@ async def get(self, dateobs=None):
                 Arrow-parseable date string (e.g. 2020-01-01). If provided, filter by
                 dateobs <= endDate
             - in: query
-              name: tag
+              name: tagKeep
               nullable: true
               schema:
                 type: string
               description: |
                 Gcn Tag to match against
+            - in: query
+              name: tagRemove
+              nullable: true
+              schema:
+                type: string
+              description: |
+                Gcn Tag to filter out
```

Comment:
We should do something similar to this with source classifications

Reply:
yeah maybe one of the undergrads can merge the logic in a future PR.